apps/percona/deployers/deploy_percona_litmus.yaml

```diff
@@ -0,0 +1,27 @@
+---
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: litmus-percona
+  namespace: litmus 
+spec:
+  template:
+    metadata:
+      name: litmus
+   
+    spec:
+      serviceAccountName: litmus
+      restartPolicy: Never
+      containers:
+      - name: ansibletest
+        image: ranjnshashank855/percona-installer
```

Comment:
The image needs to come from standard docker repos or from openebs. Also the images should be tagged.